tests/test_permissions.py

```diff
@@ -0,0 +1,1254 @@
+    ),
+    field_id('Mutation', 'createShareObject'): TestData(tenant_perm=MANAGE_SHARES, resource_perm=CREATE_SHARE_OBJECT),
+    field_id('Mutation', 'createTableDataFilter'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=CREATE_TABLE_DATA_FILTER
+    ),
+    field_id('Mutation', 'createTerm'): TestData(
+        tenant_perm=MANAGE_GLOSSARIES, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'createWorksheet'): TestData(
+        tenant_perm=MANAGE_WORKSHEETS, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteAttachedMetadataForm'): TestData(
+        tenant_ignore=IgnoreReason.BACKPORT, resource_perm=ATTACH_METADATA_FORM
+    ),
+    field_id('Mutation', 'deleteCategory'): TestData(
+        tenant_perm=MANAGE_GLOSSARIES, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteConnectionGroupPermission'): TestData(
+        tenant_ignore=IgnoreReason.BACKPORT, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteDashboard'): TestData(tenant_perm=MANAGE_DASHBOARDS, resource_perm=DELETE_DASHBOARD),
+    field_id('Mutation', 'deleteDataPipeline'): TestData(tenant_perm=MANAGE_PIPELINES, resource_perm=DELETE_PIPELINE),
+    field_id('Mutation', 'deleteDataPipelineEnvironment'): TestData(
+        tenant_perm=MANAGE_PIPELINES, resource_perm=UPDATE_PIPELINE
+    ),
+    field_id('Mutation', 'deleteDataset'): TestData(tenant_perm=MANAGE_DATASETS, resource_perm=DELETE_DATASET),
+    field_id('Mutation', 'deleteDatasetStorageLocation'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=DELETE_DATASET_FOLDER
+    ),
+    field_id('Mutation', 'deleteDatasetTable'): TestData(
+        tenant_perm=MANAGE_DATASETS, resource_perm=DELETE_DATASET_TABLE
+    ),
+    field_id('Mutation', 'deleteEnvironment'): TestData(
+        tenant_perm=MANAGE_ENVIRONMENTS, resource_perm=DELETE_ENVIRONMENT
+    ),
+    field_id('Mutation', 'deleteGlossary'): TestData(
+        tenant_perm=MANAGE_GLOSSARIES, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteMetadataForm'): TestData(
+        tenant_perm=MANAGE_METADATA_FORMS, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteMetadataFormField'): TestData(
+        tenant_perm=MANAGE_METADATA_FORMS, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteMetadataFormVersion'): TestData(
+        tenant_perm=MANAGE_METADATA_FORMS, resource_ignore=IgnoreReason.NOTREQUIRED
+    ),
+    field_id('Mutation', 'deleteNetwork'): TestData(tenant_perm=MANAGE_ENVIRONMENTS, resource_perm=DELETE_NETWORK),
+    field_id('Mutation', 'deleteNotification'): TestData(
+        tenant_ignore=IgnoreReason.APPSUPPORT, resource_ignore=IgnoreReason.APPSUPPORT
+    ),
+    field_id('Mutation', 'deleteOmicsRun'): TestData(
+        tenant_perm=MANAGE_OMICS_RUNS, resource_ignore=IgnoreReason.NOTREQUIRED
```

Comment:
Delete omics runs is looping over delete_omics_run which checks DELETE_OMICS_RUN